### PR TITLE
request cert-manager 2.7.0 for next aws releases

### PR DESCRIPTION
**Pull request by ubergesundheit**
Next releases AWS releases should include cert-manager 2.7.0

### CODE REVIEW

File: aws/requests.yaml [hunk 1]
```diff
@@ -16,6 +16,11 @@ releases:
         - name: coredns
           version: ">= 1.4.0"
           issue: https://github.com/giantswarm/giantswarm/issues/13771
+    - name: "> 14.1.1 > 13.1.1"
```
Comment (njuettner) [inline aws/requests.yaml:19]: Does the order of version matter?
`"> 14.1.1 > 13.1.1"` vs `"> 13.1.1 > 14.1.1"`

Comment (ubergesundheit) [reply aws/requests.yaml:19]: No idea. Who could know? For funsies lets reverse this and push again to see what CI thinks

Comment (ubergesundheit) [reply aws/requests.yaml:19]:
```suggestion
    - name: "> 13.1.1 > 14.1.1"
```

**Review by njuettner — APPROVED** [review]

File: aws/requests.yaml (hunk 1 above)
Comment (njuettner) [reply aws/requests.yaml]: Not sure I only saw it some lines below the order was different 🙂 .

Comment (ubergesundheit) [reply aws/requests.yaml]: Its parsed by this function https://github.com/giantswarm/releases/blob/master/releases_test.go#L198 (called https://github.com/giantswarm/releases/blob/master/releases_test.go#L103)

**Comment by ubergesundheit** [comment]
Thanks!

**Pull request merged**